solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+import DoubleIcon from "./DoubleIcon"
+import * as Icons from "./Icons"
+import { SubmitButton } from "./Button"
+import Card from "./Card"
+
+const LiquidityRewardCard = ({ title, MainIcon, SecondaryIcon, viewPoolLink }) => {
+  return (
+    <Card className={"tile"}>
+      <div className={"liquidity__card-title-section"}>
```

Comment:
At the top in `Card` component should be `liquidity-card` then `liquidity-card__title`(I think the section word is unnecessary here) and so on. According to the BEM methodology.

Reply:
Fixed in https://github.com/keep-network/keep-core/pull/2216/commits/ff520422a7149ced8fa83e2c17182dd05fa9f156